api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -135,6 +136,8 @@ message ExtAuthz {
   //
   // When this field is true, Envoy will include the peer X.509 certificate, if available, in the
   // :ref:`certificate<envoy_v3_api_field_service.auth.v3.AttributeContext.Peer.certificate>`.
+  //
+  // This field MUST NOT be set when the filter is configured on an upstream filter chain.
```

Comment:
I think this is something that may be of interest to ext_authz server. Is there a specific reason why this is not going to work for upstream ext_authz?

Reply:
It, and other things, seem to require the downstream connection, which is not available on upstream filters. source/extensions/filters/common/ext_authz/check_request_utils.cc

Reply:
It seems like there should be some way to get it. Maybe look at how xfcc works, because that includes the cert in upstream requests.

Reply:
XFCC is being implemented in the HttpConnectionManager which has references to the downstream connection. The upstream filter chain, at the moment, does not.

I am reading alyssawilk@'s comment in https://github.com/envoyproxy/envoy/pull/29173#discussion_r1307782625 as saying that we could potentially plumb this into upstream requests. It being that this isn't currently being done, I'm proposing disallowing this combination until such time as downstream connection fields are available on the upstream streamInfo.